src/model/reviews.model.js

```diff
@@ -0,0 +1,41 @@
+import { Schema, model } from "mongoose";
+
+const reviewSchema = new Schema({
+  comment: {
+    type: String,
+    required: [true, "El comentario es requerido"],
+  },
+  rating_cleanliness: {
+    type: int,
+    required: [true, "La calificación de limpieza es requerida"],
+  },
+  rating_staff: {
+    type: int,
+    required: [true, "La calificación del personal es requerida"],
+  },
+  rating_facilities: {
+    type: int,
+    required: [true, "La calificación de las instalaciones es requerida"],
+  },
+  tp_status: {
+    type: String,
+    required: [true, "El estado es requerido"],
+    enum: ["ACTIVE", "INACTIVE"],
+  },
```

Comment:
Deberia tener por defecto `ACTIVE` 
```suggestion
  tp_status: {
    type: String,
    required: [true, "El estado es requerido"],
    enum: ["ACTIVE", "INACTIVE"],
    default: "ACTIVE"
  },
```